crates/evm/src/lib.rs

```diff
@@ -31,6 +31,132 @@ pub mod system_calls;
 /// test helpers for mocking executor
 pub mod test_utils;
 
+/// Builder for creating an EVM with a database and environment.
+#[derive(Debug)]
+pub struct RethEvmBuilder<DB: Database, EXT> {
```

Comment:
this needs a few more docs, that this is a wrapper for revm evmbuilder

Reply:
fixed